### PR TITLE
Enable Windows build and tests

### PR DESCRIPTION
**Pull request by chambbj**
@conda-forge-admin, please rerender

### CODE REVIEW

**Comment by chambbj** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,8 +12,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  skip: True  # [win]
+  number: 1
+  skip: True  # [win and py!=36]
   features:
     - vc14  # [win]
```
Comment (isuruf) [inline recipe/meta.yaml:19]: You need `- python  # [win]` in build for the python/vc matrix

Comment (chambbj) [reply recipe/meta.yaml:19]: Yes, just caught that. Thanks @isuruf!

**Comment by chambbj** [comment]
@conda-forge-admin, please rerender

**Comment by chambbj** [comment]
@conda-forge-admin, please rerender

**Comment by chambbj** [comment]
@conda-forge-admin, please rerender

**Comment by chambbj** [comment]
@conda-forge-admin, please rerender

**Pull request merged**